CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
+- Added non allocating SentrySdk.ConfigureScope overload ([#4244](https://github.com/getsentry/sentry-dotnet/pull/4244))
```

Comment:
suggestion: rephrase the changelog entry slightly:

```suggestion
- Added non-allocating `ConfigureScope` and `ConfigureScopeAsync` overloads ([#4244](https://github.com/getsentry/sentry-dotnet/pull/4244))
```

- I believe "non-allocating" is a hyphenated compound
- Remove `SentrySdk` since we also add these overloads to Hub and ScopeManager
- Also mention the `Async` variant